tests/plugins/colorbutton/manual/customcolors.html

```diff
@@ -0,0 +1,7 @@
+<textarea id="editor" name="editor">Test text.</textarea>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		colorButton_colors: 'Not blue/F00,00F'
```

Comment:
Maybe restrict toolbar to only one colorbutton? ATM there are two buttons: for background and text.